<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlvSivu</class>
 <widget class="QWidget" name="AlvSivu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>962</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
     <item>
      <widget class="QGroupBox" name="ilmoitusGroup">
       <property name="title">
        <string>Uusi ALV-ilmoitus</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="kaudelleLabel">
            <property name="text">
             <string>Alv-kausi</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="kaudelleCombo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="alkaaLabel">
            <property name="text">
             <string>Alv-ilmoitus alkaa</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="alkaaEdit">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="paattyyLabel">
            <property name="text">
             <string>Alv-ilmoitus päättyy</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="paattyyEdit">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Eräpäivä</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="erapaivaLabel">
            <property name="text">
             <string>xxx</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="tilitaNappi">
          <property name="text">
           <string>&amp;Tee alv-ilmoitus</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/verotilitys.png</normaloff>:/pic/verotilitys.png</iconset>
          </property>
          <property name="shortcut">
           <string>F11</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="yhteenvetoGroup">
       <property name="title">
        <string>Yhteenvetoilmoitus</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="yhteenvetoLabel">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="yhteenvetoButton">
          <property name="text">
           <string>Tee &amp;yhteenvetoilmoitus</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/eu.png</normaloff>:/pic/eu.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Annetut ALV-ilmoitukset</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="ilmoituksetView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="tilitysNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Näytä tilitys</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/print.png</normaloff>:/pic/print.png</iconset>
       </property>
       <property name="shortcut">
        <string>F9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="poistaTilitysNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Poista tilitys</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/roskis.png</normaloff>:/pic/roskis.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ilmoitinNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ilmoitinaineisto</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/vie.png</normaloff>:/pic/vie.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
